go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/LukeJelly/Cell-Compete-Go
 
 go 1.13
+
+require github.com/stretchr/testify v1.5.1
```

Comment:
This gets added automatically just by importing the library and building the project.

`go.mod` contains the specific library we want to import and the version we imported.